docs/bonding-intro.md

```diff
@@ -61,7 +61,7 @@ SRTSOCKET sock = srt_create_socket();
 The listener needs to bind it first (note: simplified code):
 
 ```
-sockaddr_in sa = CreateAddrInet("0.0.0.0:5000");
+sockaddr_in sa = CreateAddr("0.0.0.0", 5000);
```

Comment:
Note that `CreateAddr` returns `sockaddr_any`, not `sockaddr_in`.

Reply:
Right, missed that!